processor/src/main/java/org/mobilitydata/gtfsvalidator/processor/Analyser.java

```diff
@@ -0,0 +1,144 @@
+package org.mobilitydata.gtfsvalidator.processor;
```

Comment:
We need the Apache license header on all classes with copyright Google 2020.

Reply:
Done.